### PR TITLE
[8.0] Add purchase_order_line_description module

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: purchase_order_line_description/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Alex Comba <email.address@example.org>
```
Comment (eLBati) [inline purchase_order_line_description/README.rst]: This is not your email :smirk:

**Comment by eLBati** [comment]
1 remark, but :+1:

File: purchase_order_line_description/README.rst (hunk 1 above)
Comment (tafaRU) [inline purchase_order_line_description/README.rst]: @eLBati, fixed. Thanks!

**Comment by andhit-r** [comment]
@tafaru 

I tested on runbot. But i still can change description manually

purchase_description

File: purchase_order_line_description/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Purchase order line description
+===============================
+
+This module allows to use only the product purchase description on the purchase
+order lines.
```
Comment (eLBati) [inline purchase_order_line_description/README.rst:10]: @tafaRU please add the comparison with standard behaviour, to highlight the purpose of the module

**Comment by eLBati** [comment]
@andhit-r the purpose of this module is not to prevent user to manually change the description, but to use product description instead of CODE + DESCRIPTION

@tafaRU correct me if I'm wrong and please improve README https://github.com/OCA/purchase-workflow/pull/208#discussion_r83367764

**Review by pedrobaeza — APPROVED** [review]
It's working as expected

**Pull request merged**